WORKSPACE

```diff
@@ -38,17 +38,17 @@ bind(
 # 2. Update .bazelversion, envoy.bazelrc and .bazelrc if needed.
 #
 # envoy-wasm commit date: Jan 23 2020
-ENVOY_SHA = "59be1b34f15d23821d8ad22c75e6ad8086334398"
+ENVOY_SHA = "277a86a6aaf2f4a22f9060969fb0cca"
```

Comment:
I guess this should be `277a86a6aaf2f4a22f9060969fb0cca4ab8d7a79`?

Reply:
Fixed.